networkx/algorithms/tests/test_summarization.py

```diff
@@ -1,9 +1,9 @@
 """
-Unit tests for dedensification/densification
+Unit tests for dedensification and graph summarization
 """
 
 import networkx as nx
-from networkx.algorithms.summarization import dedensify
+from networkx.algorithms import summarization
```

Comment:
Doesn't `nx.dedensify` work?  Do we need to import the summarization module explicitly?
`nx.snap_aggregation` should work too (any function to put into `__all__`).